### ISSUE TITLE
Wrong quality with defaults loot values?

### ISSUE THREAD

**Issue by piotrch** [thommcgrath/Beacon#70]
Hi,
Please correct me if I'm wrong, but when I import vanilla Island settings posted on the beaconapp.cc/help/  then export it to my game.ini there should be no change in loot at all? If so, something's wrong because I keep getting only primitive items (same quality as crafted from engrams).

**Comment by thommcgrath** [comment]
No, not exactly. Beacon is specifically designed to counteract the loot source multipliers. So the act of bringing the default table into Beacon ends up nuking the quality. The author of those files would need adjust the settings.

**Comment by piotrch** [comment]
Ok got it. Changed manually quality in each item set of each loot source and it looks well now. Too bad it is hard to find exact match to original values but I understand it is a complicated system. Personally I miss two options in the program:
- finding all item sets containing selected item
- setting min/max quality of all item sets contained in selected loot source (or better shift-selecting many item sets and mass setting their min/max quality)
